RELEASE_NOTES.md

```diff
@@ -81,19 +81,23 @@ you are strongly advised to take a backup first.
   along with the traditional color-picker for manual sampling of neutral colors.
 
   Color calibration internally implements gamut-mapping and gamut-clipping,
-  which attempts to preserve the hue and luminance of pixels while avoiding imaginary and
-  out-of-gamut colors at the beginning of the pipeline, to improve the robustness of
+  which attempt to preserve the hue and luminance of pixels while avoiding imaginary and
+  out-of-gamut colors at the beginning of the pipeline, in order to improve the robustness of
```

Comment:
If 'which' refers back to 'gamut-mapping' and 'gamut-clipping' (so there is more than one subject), the verb should be in plural form.